tests/framework/decorators.py

```diff
@@ -2,6 +2,54 @@
 # SPDX-License-Identifier: Apache-2.0
 """Module for declaring decorators used throughout integration tests."""
 
+import time
```

Comment:
nit: Can you rephrase the commit message so you don't need to have dots in your commit message?
Suggestion: "CI: move decorates to dedicated file"

Reply:
Rephrased it.